### PR TITLE
reading content added for Chapter 2

### PR DESCRIPTION
**Pull request by speudusa**
- [x] Reading 3 sections
- [x] Concept Checks for each section

### CODE REVIEW

File: content/data-types/exercises/_index.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+   public static void main(String[] args) {
+      Scanner input = new Scanner(System.in);
+      System.out.println("Hello, what is your name:");
+
+      String name = input.nextLine();
+      System.out.println("Hello " + name);
+   }
+}
+```
+{{% /expand %}}
```
Comment (codinglikeagirl42) [inline content/data-types/exercises/_index.md]: Suggest tabbing over to keep consistant with the code blocks from above.

File: content/data-types/exercises/_index.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: # UPDATE ANY TIME CHANGES ARE MADE
+---
+
+## Getting Started
+
+<!-- TODO: Link to chapter 1 for naming conventions-->
+
+Work on these exercises in the IntelliJ `java-web-dev-exercises` project. Create a new class for each numbered exercise. You may name the classes whatever you like, but use proper [Java Naming Conventions](Link to chapter 1) and make sure that the file name matches the class name.
```
Comment (codinglikeagirl42) [inline content/data-types/exercises/_index.md]: Suggest adding link to repo https://github.com/LaunchCodeEducation/java-web-dev-projects-17 and modify the wording to match name of repo.

File: content/data-types/reading/some-java-practice/_index.md [hunk 3]
```diff
@@ -0,0 +1,269 @@
...
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: # UPDATE ANY TIME CHANGES ARE MADE
+---
+
+Let’s move beyond our “Hello, World” example and explore a simple temperature conversion program. We want our function to convert a Fahrenheit temperature to Celsius.
+
+## Temperature Conversion
+
+1. Open the `TempConverter` file in the `java-web-dev-exercises` project in IntelliJ.
```
Comment (codinglikeagirl42) [inline content/data-types/reading/some-java-practice/_index.md]: Section needs reworking and link to project https://github.com/LaunchCodeEducation/java-web-dev-projects-17.  Also update image.
tempConversion

File: content/data-types/studio/_index.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+ public static void main(String[] args)
+```
+{{% /notice %}}
+
+## B. The `Circle` Class
+
+Add a second Java file to your program to delegate the area calculation away from the printing task.
+
+1. Add a new class in your `studios.areaofacircle` package called `Circle`.
+1, Create a method called `getArea` inside of `Circle` that takes a `Double` `radius` as its only parameter and returns another `Double`, the result of the area calculation.
```
Comment (codinglikeagirl42) [inline content/data-types/studio/_index.md]:
```suggestion
1. Create a method called `getArea` inside of `Circle` that takes a `Double` `radius` as its only parameter and returns another `Double`, the result of the area calculation.
```

File: content/data-types/studio/_index.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+   }
+   ```
+   {{% /notice %}}
+1. Back in `Area`, replace your area calculation line with a call to `Circle`.`getArea()`.
+
+{{% notice green "Tip" "rocket" %}} 
+ Check out the HelloMethods and Message example from [Static Methods](../reading/more-data-types/#static-methods) for a reference on how to use a class from another file.
+{{% /notice %}}
+
+Great work! Don't forget to save, `stage`, and `commit` your work up to GitHub.
```
Comment (codinglikeagirl42) [inline content/data-types/studio/_index.md]: Suggest to tab over to match formatting from above.

File: content/data-types/next-steps.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+originalAuthor: <no value> # to be set by page creator
+originalAuthorGitHub: <no value> # to be set by page creator
+reviewer: # to be set by the page reviewer
+reviewerGitHub: # to be set by the page reviewer
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: # UPDATE ANY TIME CHANGES ARE MADE
+---
+
+## Next Steps
```
Comment (codinglikeagirl42) [inline content/data-types/next-steps.md]: Add next steps.

File: content/data-types/reading/images/tempConverterTree.png
Comment (codinglikeagirl42) [inline content/data-types/reading/images/tempConverterTree.png:1]: Update Image to match current repo name and structure

**Review by codinglikeagirl42 — CHANGES_REQUESTED** [review]
Please review comments and update.  When done please resubmit for review.

File: content/data-types/studio/_index.md [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+---
+
+Get cozy with Java syntax by writing a console program that calculates the area of a circle based on input from the user.
+
+## Creating your class
+Since you’re still new to Java and IntelliJ, we’ll provide some extra direction the first studio.
+
+First, make a new folder, or package, to hold your studio work. Create a new package named `org.launchcode.java.studios.areaofacircle` by right-clicking (or ctrl-clicking for some Mac users) on the `src` directory in `java-web-dev-exercises` and selecting _New_ > _Package_. Be sure to enter the full name, or your package won’t be created in the correct location.
+
+Create your class in the `java-web-dev-exercises` project within the package `org.launchcode.java.studios.areaofacircle` by right-clicking/ctrl-clicking on the `studios.areaofacircle` package/folder and selecting _New_ > _Java Class_. Enter the name Area. Select the option to add the file to Git when the window appears.
```
Comment (codinglikeagirl42) [inline content/data-types/studio/_index.md]: Suggestion to simplify by making instructions an ordered list like in exercises. Please update based on the new repo name, instructions should match how the studio appears in solution.

File: content/data-types/exercises/_index.md (hunk 1 above)
Comment (speudusa) [reply content/data-types/exercises/_index.md]: i like the solution the way it is.  for me, it looks different than the instructions. this tells me that it's not an instruction.

File: content/data-types/studio/_index.md (hunk 7 above)
Comment (speudusa) [reply content/data-types/studio/_index.md]: check!

File: content/data-types/studio/_index.md (hunk 5 above)
Comment (speudusa) [reply content/data-types/studio/_index.md]: check!

File: content/data-types/exercises/_index.md (hunk 2 above)
Comment (speudusa) [reply content/data-types/exercises/_index.md]: name has been updated - we are using the java-web-dev-projects repo

File: content/data-types/reading/some-java-practice/_index.md (hunk 3 above)
Comment (speudusa) [reply content/data-types/reading/some-java-practice/_index.md]: new repo - new screen shots

File: content/data-types/next-steps.md (hunk 6 above)
Comment (speudusa) [reply content/data-types/next-steps.md]: added documentation links

File: content/data-types/reading/images/tempConverterTree.png
Comment (speudusa) [reply content/data-types/reading/images/tempConverterTree.png:1]: agree! and done.

File: content/data-types/studio/_index.md (hunk 4 above)
Comment (speudusa) [reply content/data-types/studio/_index.md]: good catch! thanks

**Review by jwoolbright23 — APPROVED** [review]
The changes all look great. Approving.

**Pull request merged**